### PR TITLE
Eviction service performance test

### PR DESCRIPTION
**Pull request by tbak**

### CODE REVIEW

File: titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java [hunk 1]
```diff
@@ -153,10 +172,21 @@ private Subscription observeJob() {
 
         return context.getJobManagementClient()
                 .killTask(taskId, false)
-                .onErrorResumeNext(e -> Observable.error(new IOException("Failed to kill task " + taskId + " of job " + name, e)))
+                .onErrorResumeNext(e -> Observable.error(new IOException(String.format("Failed to kill task %s  of job %s: error=%s", taskId, name, e.getMessage()), e)))
```
Comment (corindwyer) [inline titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java]: extra space after `%s`. also, shouldn't the jobId also be in here. There could be hundreds of jobs with the same name.

File: titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java [hunk 2]
```diff
@@ -153,10 +172,21 @@ private Subscription observeJob() {
...
     }
 
+    @Override
+    public Mono<Void> evictTask(String taskId) {
+        Preconditions.checkState(doRun, "Job executor shut down already");
+        Preconditions.checkNotNull(jobId);
+
+        return context.getEvictionServiceClient()
+                .terminateTask(taskId, "Simulator")
+                .onErrorResume(e -> Mono.error(new IOException(String.format("Failed to evict task %s  of job %s: error=%s", taskId, name, e.getMessage()), e)))
```
Comment (corindwyer) [inline titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java]: same as above

File: titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java (hunk 1 above)
Comment (tbak) [reply titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java]: Fixed

File: titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java (hunk 2 above)
Comment (tbak) [reply titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java]: Fixed

File: titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java (hunk 1 above)
Comment (tbak) [reply titus-testkit/src/main/java/com/netflix/titus/testkit/perf/load/job/AbstractJobExecutor.java]: The name field is unique (it is not application name).

**Pull request merged**